tests/src/Endpoint/IdentifierEndpointTest.php

```diff
@@ -29,6 +29,7 @@ class IdentifierEndpointTest extends TestCase
      *
      * @covers \OpenEuropa\CdtClient\Endpoint\IdentifierEndpoint
      * @covers \OpenEuropa\CdtClient\Endpoint\EndpointBase
+     * @covers \OpenEuropa\CdtClient\Http\Rest
```

Comment:
I decided not to introduce any dedicated test for the Rest class. It is covered completely in other tests.